Update invalid styling for all remaining elements

mylesw27 opened the pull request:
This PR builds off of #11489 and adds this updating styling for invalid and missing inputs to all remaining pl-elements that display an invalid status. Please see the screenshots of the updated elements below. 

Invalid input in the question panel:
![Screenshot 2025-04-15 at 10 55 35 AM](https://github.com/user-attachments/assets/206a7cce-f76e-4282-9113-cc0f080f7c95)

Invalid input in the submission panel:
![Screenshot 2025-04-15 at 10 55 41 AM](https://github.com/user-attachments/assets/3decaaa6-8723-49d6-913a-c523eb6118c6)

Missing input in the submission panel:
![Screenshot 2025-04-15 at 11 20 48 AM](https://github.com/user-attachments/assets/7dd24934-7e8a-4036-bb50-09d69c54cb54)

Missing input message:
![Screenshot 2025-04-15 at 11 29 23 AM](https://github.com/user-attachments/assets/6e790ff9-8543-4f4f-bf2b-7292bd6044ad)

eliotwrobson commented:
> Several of these elements (`pl-checkbox`, `pl-drawing`, `pl-dropdown`, `pl-matching`, `pl-matrix-component-input` and `pl-matrix-input`) did not have any invalid feedback in the question panel but did have it in the submission panel. Was there a specific reason that we did not have an invalid message in the question panel for these? I did not add it but can do so if we decide we want it on those.

I don't think there was any specific reason, but it would be good to include this for the commonly used elements like checkbox and drawing. More likely is that the invalid input just never came up for these elements because it's a pretty uncommon situation.

mylesw27 commented on apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache:
```diff
@@ -64,17 +75,23 @@
+            </a>
+        </span>
     {{/parse_error}}
     {{#missing_input}}
```
Good catch! Updated the missing input here in [e7d159b](https://github.com/PrairieLearn/PrairieLearn/pull/11540/commits/e7d159b18feaec93582a22e2aaf23b1add81e541).

mylesw27 commented:
We received feedback that the message in the invalid feedback in the question panel can be misleading. We removed the message in #11592 and I have updated this current PR to reflect this change in [21bc6f2](https://github.com/PrairieLearn/PrairieLearn/pull/11540/commits/21bc6f2c76da873f425697f484a0cc10cd412a45). 

The question panel will now look like the following:
<img width="468" alt="Screenshot 2025-03-19 at 2 54 27 PM" src="https://github.com/user-attachments/assets/9cf67b0f-2563-4af3-a161-690f45ec1dd4" />

nwalters512 commented on apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.mustache:
```diff
@@ -64,25 +72,43 @@
-        $)$
+        <span class="small" style="white-space: nowrap">
+            <span class="badge text-bg-dark border">
+                Missing Input 
```
Nit, can we update all these to use sentence case ("Missing input") instead? Can you also include a screenshot of how one of these missing input messages look in the PR description for posterity?

mylesw27 replied:
Updated the missing inputs to use sentence case in [76ea857](https://github.com/PrairieLearn/PrairieLearn/pull/11540/commits/76ea857e12962bd5d131714d32120d3ea467cafa). 

Also, added screenshots including the missing input messages to the PR description.

nwalters512 reviewed (approved):
@mylesw27 I didn't test these myself; I assume you've done so, both for the invalid submission case and for missing input.
